examples/object_detection/layers/box_utils.py

```diff
@@ -188,7 +189,10 @@ def forward(ctx, boxes, scores, threshold, top_k=200):
             return torch.tensor([], dtype=torch.int), torch.tensor(0)
         if scores.dim() == 1:
             scores = scores.unsqueeze(1)
-        keep = EXTENSIONS.nms(torch.cat((boxes, scores), dim=1), threshold, top_k)
+        if boxes.device.type == 'cpu':
```

Comment:
I'd use `is_cuda` method for the Tensor. Please also add type hints for the function to make things clear

Reply:
Done